### PR TITLE
Issue #12: Add slack link

### PR DESCRIPTION
**Pull request by mugdha-adhav**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -19,8 +19,8 @@ home = [ "HTML", "RSS", "JSON"]
   themeVariant = "blue"
 
 [[menu.shortcuts]]
-name = "<i class='fab fa-fw fa-slack-hash'></i> Add to Slack"
-url = "https://slack.com/oauth/authorize?client_id=12637824912.569770799590&scope=commands,bot"
+name = "<i class='fab fa-fw fa-slack'></i> Community Slack"
+url = "https://botkube-slack.herokuapp.com"
```
Comment (ssudake21) [inline config.toml]: Change this to join.botkube.io
Wait for @vishal-biyani to change DNS mapping.

Comment (mugdha-adhav) [reply config.toml]: sure.

Comment (ssudake21) [reply config.toml]: Currently redirection is working. Lets go ahead with redirection for now

Comment (mugdha-adhav) [reply config.toml]: Made all requested changes.

**Pull request merged**